src/Screens/Configuration/ConfigurationStackNavigator.tsx

```diff
@@ -9,7 +11,7 @@ import { ConfigurationStackParamList } from './types';
 const Stack = createNativeStackNavigator<ConfigurationStackParamList>();
 
 const ConfigurationNavigator = () => (
-  <Stack.Navigator initialRouteName="ConfigurationScreen">
+  <Stack.Navigator initialRouteName="ConfigurationScreen" screenOptions={{ headerShown: false }}>
```

Comment:
What was on the header? Should we hide it?

Reply:
Actually I don't know... In my first commit for this PR there was no "screenOptions = {{headerShown: false}}", but I think if we don't have that it will will show a white stripe at the top of the screen. Could you make this test, please?

Reply:
I agree, but since we had already started the task, Seabra and I did it together, I committed and then he merged this commit into his branch, I don't know if it will be a problem. But maybe we can leave this way, just for the sake of ease kkk if you agree.

Reply:
I was testing and noticed something, did you merge or rebase your branch with the current develop? If you didn't, can you do it please?

Reply:
I think my branch is up to date... What happened when you tried to run?

![image](https://user-images.githubusercontent.com/57528343/110179169-5c55a980-7de6-11eb-83f0-5f07849ea5cc.png)

Reply:
I think we shouldn't hide it, because it's used in all config path, maybe change only on Password Stack